### PR TITLE
Add GnuPG and its dependencies

### PR DESCRIPTION
**Pull request by livingsilver94**

### CODE REVIEW

File: dev-utils/docbook/stone.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+# but it collects all the XML schemas installed. We shouldn't
+# be shipping a catalog file with this package.
+# If a different requirement arises, we should develop
+# a post-install trigger to populate the existing vendored catalog.
+environment : |
+    # Why does boulder `cd` into /ent directory by itself?
+    cd ..
+    export xslVersion=1.79.2
+setup       : |
+    ls -lah
```
Comment (ikeycode) [inline dev-utils/docbook/stone.yml:29]: any reason this is here?

File: libs/libsecret/stone.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+#
+# SPDX-FileCopyrightText: © 2020-2023 Serpent OS Developers
+#
+# SPDX-License-Identifier: Zlib
+#
+name        : libsecret
```
Comment (ikeycode) [inline libs/libsecret/stone.yml:6]: Should this be in the GNOME repo?

File: dev-utils/docbook/stone.yml (hunk 1 above)
Comment (livingsilver94) [reply dev-utils/docbook/stone.yml:29]: Yes. Debug. Which isn't required anymore xP I'll nuke it.

**Comment by ikeycode** [comment]
per chat on matrix, we MAY end up shifting this to `gnome/` if we find it GNOME specific, but our thinking right now is that its more on the XDG end of the spectrum.

**Pull request merged**